### ISSUE TITLE
Tabnabbing in OWTF Dashboard

### ISSUE THREAD

**Issue by amanpro30** [owtf/owtf#1131]
## Expected Behavior

Tabnabbing in the help menu in OWTF Dashboard. Since tabnabbing can only happen in older version of firefox and chrome. New versions of browser have already fixed this issue. So there is a risk when OWTF is going global and some victims still may use the older version of browser.

## Possible Solution

Adding rel="noopener noreferrer" directive in all the anchor tags will fix this issue

## Steps to Reproduce (for bugs)

Ẉhen Clicking on any of the links in the help menu will open a new window.

![image](https://user-images.githubusercontent.com/37950602/114305767-2bf4df80-9af7-11eb-9bb9-55cd659ceaa4.png)
The Window interface's opener property returns a reference to the window that opened the window, either with open(), or by navigating a link with a target attribute.
In other words, if window A opens window B, B.opener returns A.

![image](https://user-images.githubusercontent.com/37950602/114305677-df110900-9af6-11eb-8da8-ff356951c593.png)

## Your Environment

* Installation method used:
* Version/branch used:
* Operating System and version (like Kali, Debian, ArchLinux, etc): Firefox 78.0.2
* pip/setuptools version:
* sudo access:

**Comment by amanpro30** [comment]
I will like to work on this issue.
